html/script.js

```diff
@@ -2159,7 +2159,9 @@ function displayPhoto() {
         return;
     }
     let photos = SelectedPlane.psAPIresponse["photos"];
-    if (!photos || photos.length == 0) {
+    console.log(photos)
+    if (!photos || photos.length === 0 || photos.size === 0) {
+        setPhotoHtml("<p>No picture found!</p>");
```

Comment:
"no picture found" is ugly, just no picture is much nicer ... that's why i chose it like that.

Reply:
I suppose it would be nice to let the user know .... let me think about it :) and thanks for looking at the code, never hurts.